_posts/2015-07-30-decoupling.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Decoupling
+categories: future-plans community
+---
+
+We are in the process of getting rid of submodules and separating our code into
+smaller repositories. Why? And why should developers and users care?
+
+The OME development workflow has been following a single, integrated approach
+for the last couple of years. This means that all components of the OME stack,
+including OMERO and Bio-Formats are versioned globally and released
+simultaneously. The only notable exception to this rule is the OME Data Model
+which is released on its own release cycle to be consumed by major versions of
+Bio-Formats and OMERO.
+
+While such a global approach can simplify both internal and external
+communication to a certain extent, it does have several limitations for
+developers and users. For example, the increasingly-frequent  security
+vulnerabilities (see the recent blog post about the Java security issue)
+usually require one of our software components to be released with a very fast
+turnaround without modifying other components. A second example is the release
+of urgent bug fixes or support for new file formats in Bio-Formats. In the
+current approach, these updates are held back until a full release of OMERO
+and Bio-Formats is ready-- delaying access and use by a very large community.
```

Comment:
double dash doesn't format nicely, I think you need to remove the space before 'delaying' to get an em-dash, or you could just use a comma since those are really a replacement for a bracketed section anyway.